internal/providers/github/app/app.go

```diff
@@ -82,11 +85,33 @@ func NewGitHubAppProvider(
 		},
 	}
 
-	tc.Transport, err = metrics.NewDurationRoundTripper(tc.Transport, db.ProviderTypeGithub)
+	transport, err := metrics.NewDurationRoundTripper(tc.Transport, db.ProviderTypeGithub)
 	if err != nil {
 		return nil, fmt.Errorf("error creating duration round tripper: %w", err)
 	}
 
+	// If $MINDER_LOG_GITHUB_REQUESTS is set, wrap the transport in a logger
+	// to record all calls and responses to from GitHub:
+	if os.Getenv("MINDER_LOG_GITHUB_REQUESTS") != "" {
+		transport = &loghttp.Transport{
+			Transport: transport,
+			LogRequest: func(req *http.Request) {
+				zerolog.Ctx(context.Background()).Info().
+					Str("type", "REQ").
+					Str("method", req.Method).
```

Comment:
should we print the rate limit headers too?

Reply:
I've addressed the above comment, i'll add support for the headers in a follow up